### DISCUSSION TITLE
Is it possible to use this fill in the original title for shows?

### DISCUSSION THREAD

**Discussion by Eagle1337** [meisnate12/Plex-Meta-Manager#12] [Q&A] [answered]
It's one thing hama won't do.

**Comment by meisnate12** [comment]
I thought I could but the plex API doesn't allow you to so I'm currently trying to find a workaround follow #3 for updates

**Reply by Soitora** [reply]
Kinda stupid that it doesn't allow you, I even made a thread about it last year, and back then I said it has almost been 10 years since first reported as an issue: https://forums.plex.tv/t/still-unable-to-set-original-title-through-an-agent/655214

**Comment by meisnate12** [comment]
i figured this out an update will be out later today

**Comment by meisnate12** [comment, marked as answer]
it is now possible using the `original_title` [Metadata Attribute](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Metadata-Attributes)
